python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -431,7 +431,7 @@ def test_dist_training_succeeds(self) -> None:
                 )
                 self.assertEqual(
                     expected,
-                    dist._run_distributed_training(dist._run_training_on_pytorch_file, "..."),
+                    dist._run_distributed_training(dist._run_training_on_pytorch_file, "...", None),
```

Comment:
`_run_distributed_training` is an internal interface. 
For internal interface, I prefer to set less default argument to prevent careless code error (e.g. unaware of default argument being set)